xbmc/guilib/guiinfo/PlayerGUIInfo.cpp

```diff
@@ -111,16 +109,14 @@ std::string CPlayerGUIInfo::GetCurrentSeekTime(TIME_FORMAT format) const
   if (format == TIME_FORMAT_GUESS && GetTotalPlayTime() >= 3600)
     format = TIME_FORMAT_HH_MM_SS;
 
-  const auto& components = CServiceBroker::GetAppComponents();
-  const auto appPlayer = components.GetComponent<CApplicationPlayer>();
   return StringUtils::SecondsToTimeString(
-      g_application.GetTime() + appPlayer->GetSeekHandler().GetSeekSize(), format);
+      g_application.GetTime() + m_appPlayer->GetSeekHandler().GetSeekSize(), format);
 }
 
 std::string CPlayerGUIInfo::GetSeekTime(TIME_FORMAT format) const
 {
-  const auto& components = CServiceBroker::GetAppComponents();
-  const auto appPlayer = components.GetComponent<CApplicationPlayer>();
+  const auto appPlayer = m_appPlayer;
```

Comment:
_can_ be removed for two subsequent calls

Reply:
Ha, right. Thanks.